R/fmelt.R

```diff
@@ -86,18 +85,18 @@ measurev = function(fun.list, sep="_", pattern, cols, multiple.keyword="value.na
     which(names(fun.list) == "")
   }
   if (length(prob.i)) {
-    stop("in measurev, ", group.desc, " must be named, problems: ", paste(prob.i, collapse=","))
+    stopf("in measurev, %s must be named, problems: %s", group.desc, brackify(prob.i))
   }
   err.names.unique = function(err.what, name.vec) {
     name.tab = table(name.vec)
     bad.counts = name.tab[1 < name.tab]
     if (length(bad.counts)) {
-      stop(err.what, " should be uniquely named, problems: ", paste(names(bad.counts), collapse=","))
+      stopf("%s should be uniquely named, problems: %s", err.what, brackify(names(bad.counts)))
```

Comment:
@tdhock FYI I think in a follow-up to this PR we'll want to revisit error messages here, it's not very translation-friendly at the moment (not a high priority)

Reply:
sure that is fine with me. what exactly does it mean to be translation-friendly?

Reply:
in this case, we would have to have translators translate sentence fragments in isolation. the way to combine those fragments as we do here in English is likely to be different in other languages.

Reply:
ok that makes sense, so in this case err.what has three possible values, "measured columns", "elements of fun.list", "... arguments to measure" which we should convert to three separate calls to stop, for example the first one becomes "measured columns should be uniquely named, problems: "

Reply:
that sounds good... full sentences are usually good